dataland-frontend/tests/component/components/general/SlideShow.cy.ts

```diff
@@ -0,0 +1,86 @@
+import TestSlideShow from "./TestSlideShow.vue";
+
+const SLIDE_DELTA = 440;
+
+describe("Component test for the SlideShow component", () => {
+  it("Check if the buttons are working", () => {
+    cy.mountWithPlugins(TestSlideShow, {}).then(() => {
+      assertSlidesPosition(slidesSelector);
+      cy.get(rightButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 1);
+      cy.get(rightButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 2);
+      cy.get(rightButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 2);
+      cy.get(leftButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 1);
+      cy.get(leftButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 0);
+      cy.get(leftButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 0);
+      cy.get(rightButtonSelector).click();
+      assertSlidesPosition(slidesSelector, 1);
+    });
+  });
+
+  it("Check if dragging is working", () => {
+    cy.mountWithPlugins(TestSlideShow, {}).then(() => {
+      assertSlidesPosition(slidesSelector);
+      dragSlideTo(genericSlideSelector, 1, leftOffset);
+      assertSlidesPosition(slidesSelector, 1);
+      dragSlideTo(genericSlideSelector, 2, leftOffset);
+      assertSlidesPosition(slidesSelector, 2);
+      dragSlideTo(genericSlideSelector, 2, leftOffset + SLIDE_DELTA);
+      assertSlidesPosition(slidesSelector, 1);
+      dragSlideTo(genericSlideSelector, 1, leftOffset + SLIDE_DELTA);
+      assertSlidesPosition(slidesSelector, 0);
```

Comment:
How do you guranatee that the point has moved far enough to switch the slide?

Reply:
`assertSlidesPosition` tests that. Or do you mean we should test the boundaries of these values?